view/frontend/web/js/view/delivery-options.js

```diff
@@ -232,7 +242,7 @@ define(
            *  matches.
            */
           window.MyParcelConfig.methods.forEach(function(method) {
-            var foundMethod = checkout.findRateByMethodCode(method);
+            const foundMethod = checkout.findRateByMethodCode(method);
```

Comment:
Waarom verander je in dit bestand alle `var` naar `const`?

Reply:
Oeps, auto fix, dit moet niet want het werkt niet in oude browsers